src/App.css

```diff
@@ -1,3 +1,77 @@
+/* Homepage */
```

Comment:
Why did you put all the css only in one file?

Reply:
Yep. A mistake/lazy move.